decompiler_agent/src/main/java/org/jrd/agent/AgentActionWorker.java

```diff
@@ -162,31 +193,60 @@ private void sendByteCode(BufferedReader in, BufferedWriter out) throws IOExcept
             out.newLine();
             out.write(encoded);
             out.newLine();
-        } catch (Exception ex) {
+        } catch (Throwable ex) {
             AgentLogger.getLogger().log(ex);
             out.write(toError(ex) + "\n");
         }
         out.flush();
     }
 
-    private void initClass(BufferedReader in, BufferedWriter out) throws IOException {
-        String fqn = in.readLine();
-        if (fqn == null) {
-            out.write(toError("No FQN provided for the init class command.") + "\n");
+    private interface Rummable {
+        void rum(String args) throws Exception;
+    }
+
+    private void executeParametrisedNoReturnCommand(BufferedReader in, BufferedWriter out, String help, Rummable rum) throws IOException {
+        String args = in.readLine();
+        if (args == null) {
+            out.write(toError(help) + "\n");
             out.flush();
             return;
         }
         try {
-            Class.forName(fqn);
+            rum.rum(args);
             out.write("DONE");
             out.newLine();
-        } catch (Exception ex) {
+        } catch (Throwable ex) {
             AgentLogger.getLogger().log(ex);
             out.write(toError(ex) + "\n");
         }
         out.flush();
     }
 
+    private void initClass(BufferedReader in, BufferedWriter out) throws IOException {
+        executeParametrisedNoReturnCommand(in, out,
+                "No FQN provided for the init class command.",
+                new Rummable() {
+                    @Override
+                    public void rum(String arg) throws Exception {
+                        Class.forName(arg);
+                    }
+                }
+        );
+    }
+
+    private void removeOverrides(BufferedReader in, BufferedWriter out) throws IOException {
+        executeParametrisedNoReturnCommand(in, out, "No regex provided for the remove override. Try .*",
+                new Rummable() {
+                    @Override
+                    public void rum(String pattern) throws Exception {
+                        int removed = provider.cleanOverrides(pattern);
+                        if (removed == 0) {
+                            throw new Exception("Nothing removed by " + pattern + ".Try OVERRIDES to see active overrides");
+                        }
+                    }
+                });
+    }
+
     private void receiveByteCode(BufferedReader in, BufferedWriter out) throws IOException {
```

Comment:
You transformed `INIT` into the new `executeParametrisedNoReturnCommand()`, why not this `OVERWRITE` too?

Reply:
Can you please elaboprate? I do not think I understand what you meen.

Reply:
You transformed the `initClass()` method into the new `executeParametrisedNoReturnCommand()` format, why not this `receiveByteCode()` method too?

Reply:
I see. Will look. Mybe I jus tdid not noticdd, or.. it have return value on top of it.

Reply:
Fair enough. on it